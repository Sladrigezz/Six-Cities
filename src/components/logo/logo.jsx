import React from 'react';
import {Link} from 'react-router-dom';

export const Logo = () => {
  return (
    <Link to={`/`}>
      <img className="header__logo" src="../img/logo.svg" alt="6 cities logo" width="81" height="41" />
    </Link>
  );
};