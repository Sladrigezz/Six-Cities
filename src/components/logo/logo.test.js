import React from 'react';
import renderer from 'react-test-renderer';
import {Logo} from './logo.jsx';

import {Router} from 'react-router-dom';
import history from '../../history/history.js';

it(`should match snapShot`, () => {
  const wrapper = renderer.create(
      <Router history={history}>
        <Logo />
      </Router>
  ).toJSON();

  expect(wrapper).toMatchSnapshot();
});
