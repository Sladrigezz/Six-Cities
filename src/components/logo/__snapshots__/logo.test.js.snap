// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapShot 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  <img
    alt="6 cities logo"
    className="header__logo"
    height="41"
    src="../img/logo.svg"
    width="81"
  />
</a>
`;
